health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -33,6 +33,14 @@
   }
 }
 
+const getMDMSUrl = (v2=false) => {
+  if(v2){
+    let url = window.globalConfigs?.getConfig("MDMS_V2_CONTEXT_PATH") || window.globalConfigs?.getConfig("MDMS_CONTEXT_PATH") || "mdms-v2";
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'url' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)